cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceAssemblyOperator.java

```diff
@@ -816,11 +970,11 @@ private KafkaRebalanceStatus buildRebalanceStatus(String sessionID, KafkaRebalan
                         }
                     }
 
-                    if (response.getJson().containsKey(CC_REST_API_SUMMARY)) {
+                    if (response.getJson().containsKey(CruiseControlRebalanceKeys.SUMMARY.getKey())) {
```

Comment:
is it possible that `response.getJson()` ever returns null? If so this check could NPE

Reply:
It is technically possible, so I will add additional condition.